CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 0.22.0
+
+### Minor Changes
```

Comment:
avoid touching the CHANGELOG.

Touch only the one changeset file you want the GHA to use

Reply:
To future self, don't run `yarn release:version`